### PR TITLE
Routes templates for pupils and teachers

### PR DESCRIPTION
**Pull request by jamesjefferies**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -113,4 +113,13 @@
     end
   end
 
+  namespace :teachers do
+    get 'schools/:school_id/chart', to: 'charts#show'
+    get 'schools/:id', to: 'schools#show', as: :school
```
Comment (jamesalmond) [inline config/routes.rb]: Why not resource routes here?

File: app/controllers/pupils/schools_controller.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class Pupils::SchoolsController < SchoolsController
```
Comment (jamesalmond) [inline app/controllers/pupils/schools_controller.rb]: I think this pattern is OK for now but I suspect we'll come across a a situation where we might want to inherit from a common `PupilsController` at some point and we'll have to decide what to do.

File: config/routes.rb [hunk 3]
```diff
@@ -113,4 +113,13 @@
     end
   end
 
+  namespace :teachers do
+    get 'schools/:school_id/chart', to: 'charts#show'
```
Comment (jamesalmond) [inline config/routes.rb]: I'm not sure what these routes are for? Do they need to be namespaced?

File: config/routes.rb (hunk 1 above)
Comment (jamesjefferies) [reply config/routes.rb]: Yep, now resource routes

File: app/controllers/pupils/schools_controller.rb (hunk 2 above)
Comment (jamesjefferies) [reply app/controllers/pupils/schools_controller.rb]: Yep, fair enough

File: config/routes.rb (hunk 3 above)
Comment (jamesjefferies) [reply config/routes.rb]: They did, but with fresh eyes, now they don't.

**Pull request closed without merging**